.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: shell
+dist: trusty
+sudo: required
+
+before_install:
+    - sudo apt-get update
+    - sudo apt-get install apt-transport-https ca-certificates curl software-properties-common
+    - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+    - sudo apt-get update || true
+    - sudo apt-get install docker-ce
+    - sudo docker run hello-world
```

Comment:
I added this to test the docker installation/execution.

In practice, it barely takes up time (2 seconds) so I'd say I like to have it.